go/teams/list.go

```diff
@@ -701,16 +701,11 @@ func parseInvitesNoAnnotate(ctx context.Context, g *libkb.GlobalContext, team *T
 				Implicit:       nil,
 				Status:         keybase1.TeamMemberStatus_ACTIVE,
 			})
-		} else if category == keybase1.TeamInviteCategory_SEITAN {
-			// no-op - do not parse seitans. We shouldn't even
-			// see them - they should all be stubbed out.
-		} else {
+		case keybase1.TeamInviteCategory_SEITAN, keybase1.TeamInviteCategory_INVITELINK:
+			// Don't show seitans or links in this view.
```

Comment:
Which view? Maybe make the function name more explicit so it is clear what this is used for.

Reply:
Not sure what a good function name here is. Do you have a suggestion?
AFAICT this is only used in `keybase team list-memberships --all` and a corresponding JSON api function.